packages/holocron/src/moduleRegistry.js

```diff
@@ -31,7 +32,7 @@ function getModuleBlockList() {
 }
 
 function registerModule(moduleName, module) {
-  modules = modules.set(moduleName, module);
+  modules = modules.set(moduleName, withHolocronErrorBoundary(module));
```

Comment:
@PixnBits do you think users should be able to opt out of this?

Reply:
Out of curiosity, why? This is a last line of defense from a module taking down a page if they don't implement their own error boundary. So if they opt out and don't implement their own, then a module would take down the page right?

Reply:
what if they want to bubble up to error boundary in root module ?

Reply:
Or give their own fallback, or give their own `onError` function.

Reply:
- [ ] Need @PixnBits feedback